isa-parser/Makefile

```diff
@@ -14,17 +14,20 @@ export E Q
 PWD:=$(shell pwd)
 include $(PWD)/../config.mk
 
-install:
+install: deps
 	$(E) "Installing.."
-	$(CABAL) install $(CABALFLAGS)
+	$(CABAL) install $(INSTALL_FLAGS)
 
-pinstall: CABALFLAGS += --enable-executable-profiling
+pinstall: CONFIG_FLAGS += --enable-executable-profiling
 pinstall: setup-config
-	$(CABAL) install $(CABALFLAGS)
+	$(CABAL) install $(CONFIG_FLAGS)
+
+deps:
+	$(CABAL) install --only-dependencies
```

Comment:
Adding it to sandbox will make `isa-parser` just behave like any other packages on hackage. Thus cabal resolves the dependencies in one go (isa-parser deemed as one of the deps). Whereas installing isa-parser will be a separate process than installing cogent, and it will resolve deps for them individually.